news/2022-09-06-triangles-voting-phase.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "triangles: Voting Phase"
+date: 2021-09-06 09:00:00 +0000
+---
+
+Mark your favourite triangle-themed maps in osu!'s latest and greatest mapping + storyboarding contest!
+
+![](https://assets.ppy.sh/contests/147/header.jpg)
+
+[A few months ago](https://osu.ppy.sh/home/news/2022-05-31-triangles), we launched a new mapping and storyboarding contest: [**triangles**](/wiki/Contests/triangles_Beatmapping_Contest), centered around osu!(lazer)'s theme song *[cYsmix](https://osu.ppy.sh/beatmaps/artists/2) - triangles*.
+
+Last month, our [mapper panel](/wiki/Contests/triangles_Beatmapping_Contest#mapper-panel) and [storyboarder panel](/wiki/en/Contests/triangles_Beatmapping_Contest#storyboarder-panel) painstakingly narrowed all submissions down to the **top 8**. Now it's your turn to choose a winner!
+
+## Vote
+
+[**Vote for your 3 favourite maps HERE!**](https://osu.ppy.sh/community/contests/147)
+
+Before you can vote, you'll need to **play one of the triangles playlists in osu!(lazer).**
+
+Access the playlists via the **play** menu on osu!(lazer) and filtering by **Featured Artists** in the top right corner. There's one playlist for each difficulty level (Easy/Normal/Hard/Insane/Expert).
+
+![](LINK)
+
+The top entries in the vote will earn these prizes:
+
+- **Community vote runner-up:** 3 months of osu!supporter
+- **Community vote winner:** 6 months of osu!supporter + *Community Pick* profile badge
+
+![Community pick profile badge](/wiki/shared/news/2022-05-26-triangles/community@2x.png)
+
+Oh, and players have a chance to earn prizes here too!
+
+When the osu!(lazer) playlists close in two weeks, the players with the best scores in each playlist will receive these prizes:
+
+- **Easy/Normal/Hard/Insane playlist leaders:** 1 month of osu!supporter each
+- **Expert playlist leader:** 1 month of osu!supporter + triangle aficionado profile badge
+
+![triangle aficionado profile badge](/wiki/shared/news/2022-05-26-triangles/aficionado@2x.png)
```

Comment:
image says "triangles aficionado" instead of "triangle aficionado", not sure which one is intended

```suggestion
![triangle aficionado profile badge](/wiki/shared/news/2022-05-26-triangles/aficionado.png)
```